snippets/dart_snippets/lib/production.dart

```diff
@@ -4,7 +4,8 @@ import 'package:breez_sdk/bridge_generated.dart';
 
 NodeConfig productionNodeConfig() {
   // ANCHOR: moving-to-production
-  // Read your Greenlight credentials from secure storage
+  // Read in your Greenlight credentials from the file 
+  // system, environment variable or build config
   Uint8List deviceKey = Uint8List(0);
   Uint8List deviceCert = Uint8List(0);
```

Comment:
These were renamed to `developerKey` & `developerCert`.

Reply:
It's not released yet though right?